### DISCUSSION TITLE
SPI connections

### DISCUSSION THREAD

**Discussion by LizShch** [TheAlbertDev/MASB#54] [Q&A] [answered]
Hello!
Just to make sure, the sensor BME280 has a cable which says "3Vo" (3.3V output from the voltage regulator), and we don't have to wire it to any terminal in the microcontroller, right?
Also, when using the SPI Mode, we can put the 0 (CPOL=0, CPHA=0) or the 3 (CPOL=1, CPHA=1) for this sensor obtaining the same result? Why?
Thanks!

**Comment by Albert-Alvarez** [comment, marked as answer]
Right, terminal `3V0` from the BME280 must not be connected to any pin of the STM32 board.

You can use the SPI mode 0 and 3 because the manufacturer has designed the BME280 sensor to be compatible with both modes.
